foo_005_rw2_crop_area_production/foo_005_rw2_crop_area_production_processing.py

```diff
@@ -0,0 +1,203 @@
+Three zipfiles were downloaded belonging to harvested area, production, and yield were downloaded through the source's API:
+
+Global production: 
+https://dataverse.harvard.edu/api/access/datafile/10120890?gbrecs=true
+Global harvest area: 
+https://dataverse.harvard.edu/api/access/datafile/10120889?gbrecs=true
+Global yield:
+https://dataverse.harvard.edu/api/access/datafile/10120888?gbrecs=true
+'''
+
+url_list = ['https://dataverse.harvard.edu/api/access/datafile/10120889?gbrecs=true',
+            'https://dataverse.harvard.edu/api/access/datafile/10120890?gbrecs=true',
+            'https://dataverse.harvard.edu/api/access/datafile/10120888?gbrecs=true']
+
+# download the data from the source
+raw_data_file = [f'{os.path.join(data_dir,os.path.basename(urllib.parse.urlparse(url).path))}.zip' for url in url_list]
+for url, file in zip(url_list, raw_data_file):
+    print('Initiating download for ', file)
+    if not os.path.isfile(file):
+        r = requests.get(url, allow_redirects=True, timeout = 10)
+        if r.status_code != 200:
+            print('Error downloading file')
+            sys.exit(1)
+        with open(file, 'wb') as f:
+            for chunk in r.iter_content(chunk_size = 8192):
+                f.write(chunk)
+
+# Unzip raw data
+# Only using six crops: maize, rice, wheat, soybean, coffee, and cotton
+# Only extracting the tif files that encompass all technologies (check source metadata)
+
+# Create a list with files of interest
+tif_list = [
+    'spam2020_v1r0_global_H_MAIZ_A.tif',
+    'spam2020_v1r0_global_H_RICE_A.tif',
+    'spam2020_v1r0_global_H_WHEA_A.tif',
+    'spam2020_v1r0_global_H_SOYB_A.tif',
+    'spam2020_v1r0_global_H_ACOF_A.tif',
+    'spam2020_v1r0_global_H_RCOF_A.tif',
+    'spam2020_v1r0_global_H_COTT_A.tif',
+    'spam2020_v1r0_global_P_SOYB_A.tif',
+    'spam2020_v1r0_global_P_WHEA_A.tif',
+    'spam2020_v1r0_global_P_RICE_A.tif',
+    'spam2020_v1r0_global_P_MAIZ_A.tif',
+    'spam2020_v1r0_global_P_ACOF_A.tif',
+    'spam2020_v1r0_global_P_RCOF_A.tif',
+    'spam2020_v1r0_global_P_COTT_A.tif',
+    'spam2020_v1r0_global_Y_MAIZ_A.tif',
+    'spam2020_v1r0_global_Y_RICE_A.tif',
+    'spam2020_v1r0_global_Y_WHEA_A.tif',
+    'spam2020_v1r0_global_Y_SOYB_A.tif',
+    'spam2020_v1r0_global_Y_ACOF_A.tif',
+    'spam2020_v1r0_global_Y_RCOF_A.tif',
+    'spam2020_v1r0_global_Y_COTT_A.tif'
+]
+# Create list to append location of files of interest
+unzipped_list = []
+# Extract files in data directory
+for index,element in enumerate(raw_data_file):
+    with ZipFile(raw_data_file[index], 'r') as zf:
+        for file in zf.namelist():
+            #print(file)
+            if os.path.basename(file) in tif_list:
+                unzipped_list.append(file)
+                zf.extract(file,data_dir)
+
+#print('unzipped_list', unzipped_list)
+# Create path to unzipped files
+raw_data_file_unzipped = [os.path.join(data_dir, os.path.basename(file)) for file in unzipped_list]
```

Comment:
The `current raw_data_file_unzipped` and `processed_data_files` would cause the error. I added the one that works for me in the comments.

`raw_data_file_unzipped = [os.path.join(data_dir, file) for file in unzipped_list]`

Reply:
Modified..